src/theory/arith/arith_poly_norm.cpp

```diff
@@ -499,15 +503,19 @@ bool PolyNorm::isArithPolyNormAtom(TNode a, TNode b)
   }
```

Comment:
You need to set ca/cb in the block above (which handles bv equality) to Rational(1).

Reply:
Nice catch!